packages/gestalt/src/ButtonToggle/GraphicButton.tsx

```diff
@@ -0,0 +1,57 @@
+import { Children, ReactElement } from 'react';
+import Avatar from '../Avatar';
+import Box from '../Box';
+import Flex from '../Flex';
+import Icon from '../Icon';
+import {
+  AvatarThumbnail,
+  IconThumbnail,
+  ImageThumbnail,
+} from '../sharedSubcomponents/thumbnailSubcomponents';
+import Text from '../Text';
+
+type Props = {
+  graphicIcon: {
+    image?: ReactElement<typeof Image>;
+    icon?: ReactElement<typeof Icon>;
+    avatar?: ReactElement<typeof Avatar>;
+  };
+  text: string;
+  textColor: 'subtle' | 'default' | 'inverse';
+};
+
+export default function CallToAction({ graphicIcon, text, textColor }: Props) {
+  return (
+    <Flex alignItems="center" direction="column" justifyContent="center">
+      <Box
+        alignItems="center"
+        dangerouslySetInlineStyle={{ __style: {} }}
```

Comment:
???

Reply:
Fixed, this was an experiment i was doing but forgot to delete the line hahah